src/core/platforms/android/androidplatformutilities.cpp

```diff
@@ -72,14 +73,17 @@ class FileCopyThread : public QThread
 void AndroidPlatformUtilities::initSystem()
 {
   // Copy data away from the virtual path `assets:/` to a path accessible also for non-qt-based libs
-  QString appDataLocation = QStandardPaths::writableLocation( QStandardPaths::AppDataLocation );
+  const QString appDataLocation = QStandardPaths::writableLocation( QStandardPaths::AppDataLocation );
   mSystemGenericDataLocation = appDataLocation + QStringLiteral( "/share" );
+
   QFile gitRevFile( appDataLocation + QStringLiteral( "/gitRev" ) );
   gitRevFile.open( QIODevice::ReadOnly );
-  QByteArray appGitRev = getIntentExtra( "GIT_REV" ).toLocal8Bit();
   QByteArray localGitRev = gitRevFile.readAll();
+  gitRevFile.close();
+  QByteArray appGitRev = getIntentExtra( "GIT_REV" ).toLocal8Bit();
   if ( localGitRev != appGitRev )
   {
+    qDebug() << "Different build git revision detected";
```

Comment:
Should we include the (first chars of) the git revs in here?

Reply:
Done